content/src/filetype.js

```diff
@@ -184,11 +183,20 @@ function wrapTurtle(doc, domain, pragmasOnly, setupScript, instrumenter) {
       maintype = 'text/javascript';
     }
   }
-  var mainscript = '<script type="' + maintype + '">\n' + seeline;
+  var mainscript = '<script type="' + maintype + '">\n';
+  if (/javascript/.test(maintype)) {
+    // Wrap javascript in a closure (coffeescript does this automatically).
+    mainscript += '(function(){\n';
```

Comment:
When the language that the student types is Javascript, will this wrap their code in a closure?  I don't think we want us to wrap student javascript in a closure

If they type CoffeeScript, we want a closure because that't what that language always does if you just have a script type=text/coffeescript; but if they type Javascript, we don't want a closure (for the same reason - if you just write script type=text/javascript, there is no automatic closure).